homeassistant/components/flexit/climate.py

```diff
@@ -64,34 +75,65 @@ def __init__(self, hub, modbus_slave, name):
         self._heating = None
         self._cooling = None
         self._alarm = False
-        self.unit = pyflexit(hub, modbus_slave)
+        self._outdoor_air_temp = None
 
     @property
     def supported_features(self):
         """Return the list of supported features."""
         return SUPPORT_FLAGS
 
-    def update(self):
+    async def async_update(self, now=None):
```

Comment:
the now=None is used in modbus because we do our own scheduling, you do not need it.